packages/loaders/src/utils/spinner-coordinates.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const strokeWidthFrames = {
+  0: 6,
+  28: 5,
+  31: 4,
+  36: 3,
+  48: 2,
+  56: 3,
+  68: 4,
+  80: 5,
+  91: 6
+};
+
+export const rotationFrames = {
```

Comment:
Why are so many more values needed here than used by https://github.com/zendeskgarden/css-components/blob/jzempel/loaders/packages/loaders/src/_keyframes.css#L114-L136?

Reply:
btw, we're seeing much smoother movement (easy to observe when slowed down in the demo) from the keyframe values under the `jzempel/loaders` branch of `css-components`. Have you tried those exact values along with your interpolation logic here?